public/components/common/modules/events.tsx

```diff
@@ -170,10 +171,11 @@ export class Events extends Component {
     if (!this.state.hasRefreshedKnownFields) {
       try {
         this.setState({ hasRefreshedKnownFields: true, isRefreshing: true });
-        await PatternHandler.refreshIndexPattern();
-
+        if(satisfyKibanaVersion('<7.11')){
```

Comment:
'<7.11' could be a constant

Reply:
Yes, it is a good idea, but I think it is not necessary. The variable name could be overcomplicated.